pallets/stake/src/lib.rs

```diff
@@ -422,8 +424,10 @@ pub trait Config: System {
 	type Event: From<Event<Self>> + Into<<Self as System>::Event>;
 	/// The currency type
 	type Currency: Currency<Self::AccountId> + ReservableCurrency<Self::AccountId>;
-	/// Blocks per round
-	type BlocksPerRound: Get<Self::BlockNumber>;
+	/// The origin for setting inflation
+	type MonetaryPolicy: EnsureOrigin<Self::Origin>;
```

Comment:
```suggestion
	type SetMonetaryPolicyOrigin: EnsureOrigin<Self::Origin>;
```

The name `MonetaryPolicy` made me expect the the trait bound to be `Get<InflationSchedule>` or something.